### PR TITLE
Cleanup TlsCredentialsOptions

### PR DESCRIPTION
**Pull request by yashykt**

### CODE REVIEW

File: src/core/lib/security/credentials/tls/grpc_tls_credentials_options.h [hunk 1]
```diff
@@ -123,7 +123,6 @@ struct grpc_tls_credentials_options
   // provider and the distributor, so we will get the distributor directly.
```
Comment (ZhenLian) [inline src/core/lib/security/credentials/tls/grpc_tls_credentials_options.h:123]: Could you please also change the comments, if it's not the case any more? Thanks!

**Review by ZhenLian — APPROVED** [review]
Just one minor comment nit. Thanks for the clean-up!

File: src/core/lib/security/credentials/tls/grpc_tls_credentials_options.h (hunk 1 above)
Comment (yashykt) [reply src/core/lib/security/credentials/tls/grpc_tls_credentials_options.h]: Done

Comment (ZhenLian) [reply src/core/lib/security/credentials/tls/grpc_tls_credentials_options.h]: Great, thanks!

**Pull request merged**